otwcode/otwarchive
# AO3-6170 Upgrade rails (and actionpack) to 5.2.6

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6170

## Purpose

Upgrades rails (and actionpack) to 5.2.6 solving our security issues.

## Testing Instructions

Run all automated tests, verify forms can be submitted.

If you want to be thorough:

- Have a form open with the current rails version before any upgrade is performed (e.g post new work)
- Upgrade the server to the new rails version
- Verify the form can be submitted without issue

Also:
- Have a form open with the new rails version (e.g post new work)
- Downgrade the server back to the previous version
- Verify the form can be submitted without issue

Both of these are possible scenarios in production during the upgrade rollout.
Note that I tried both locally in development and never encountered any issue with either.

## References

https://github.com/rails/rails/blob/5-2-stable/actionpack/CHANGELOG.md
